src/constants.js

```diff
@@ -1 +1,4 @@
+export const PROPOSAL_STATUS_ACTIVE = 'Active'
+export const PROPOSAL_STATUS_CANCELLED = 'Cancelled'
+export const PROPOSAL_STATUS_EXECUTED = 'Executed'
```

Comment:
Wondering if we should declare this constants inside the proposal-types file with the _STRING suffix?